rust/toolchain.bzl

```diff
@@ -518,7 +518,7 @@ def _rust_toolchain_impl(ctx):
         _rename_first_party_crates = rename_first_party_crates,
         _third_party_dir = third_party_dir,
     )
-    return [toolchain]
+    return [toolchain, toolchain.make_variables]
```

Comment:
So this just means any rule that depends on a `rust_toolchain` gets the make variables?

Reply:
Yep! As seen in the test.